bootstrap

```diff
@@ -177,7 +177,20 @@ if [ "${cpu_partitioning}" == "1" ]; then
     if [ "${endpoint}" == "k8s" ]; then
 	environment="container"
     fi
-    /usr/local/bin/discover-cpu-partitioning.py --environment=${environment}
+    cpu_data=$(/usr/local/bin/discover-cpu-partitioning.py --environment=${environment})
+
+    # log the data returned
+    echo -e "${cpu_data}"
```

Comment:
print on debug mode only ?

Reply:
This output is going into what is already an extremely verbose logfile where we can go to get information about a run if needed, suppressing output here would go against the way the rest of the script is written.